src/itemLogin/itemLogin.ts

```diff
@@ -9,10 +9,17 @@ export enum ItemLoginSchemaType {
   AnonymousAndPassword = 'anonymous+password',
 }
 
+export enum ItemLoginSchemaState {
+  Active = 'active', // Guests can sign in and log in
+  Disabled = 'disabled', // Guests can't sign in or log in
+  Freeze = 'freeze', // Guests can only log in
```

Comment:
Sign in means log in. 
I think the pairs are: sign in/sign up/sign out - login/register/logout
```suggestion
  Active = 'active', // Guests can register and log in
  Disabled = 'disabled', // Guests can't register or log in
  Freeze = 'freeze', // Guests can only log in
```